### ISSUE TITLE
Execution of this statement depends on table size

### ISSUE THREAD

**Issue by panasyuk** [v-may/paneco_test#3]
https://github.com/v-may/paneco_test/blob/992cf0774d4dde65975d4be2bddd9aa769d13bc0/app/models/account.rb#L24
It's better to move this calculations to background.

**Comment by v-may** [comment]
Yes, sure. As I wrote in the Skype chat, I did not have enough time to implement it :( Andrey had already asked me about the result at that time...
